Gemfile

```diff
@@ -49,6 +49,7 @@ group :development, :test do
   gem 'better_errors',              '~> 2.1', '>= 2.1.1'
   gem 'annotate',                   '~> 2.6', '>= 2.6.10'
   gem 'pry-rails',                 '0.3.4'
+  gem 'pry-nav'
```

Comment:
Is there a benefit to pry-nav over using byebug (which is already a dependency)? Looking at the pry-nav gem, it seems like they operate pretty similarly.